qdrant-landing/content/documentation/faq/qdrant-fundamentals.md

```diff
@@ -81,3 +81,35 @@ We only guarantee compatibility if you update between consecutive versions. You
 
 In case your version is older, we only guarantee compatibility between two consecutive minor versions. This also applies to client versions. Ensure your client version is never more than one minor version away from your cluster version.
 While we will assist with break/fix troubleshooting of issues and errors specific to our products, Qdrant is not accountable for reviewing, writing (or rewriting), or debugging custom code.
+
+### Which embedding models does FastEmbed support, and how does it utilize memory?
+
+You can find the list of supported models [here](https://qdrant.github.io/fastembed/examples/Supported_Models/).
+
+Memory usage in fastEmbed depends on several factors:
+1. **Size of the Text**: Everything, including the text and its vectors, is loaded into memory. As vectors are computed, RAM consumption increases accordingly.
+2. **Data Parallelism**: fastEmbed utilizes Python's multiprocessing to split large lists of strings into smaller ones and process them in parallel. While this enhances processing speed, it can also lead to higher RAM consumption.
+3. **Model Used**: Memory usage varies depending on the model used to embed your data.
+
+For optimal performance and memory management, consider these factors when using fastEmbed.
+
+### To ensure all shards are replicated to the listener node, can I use the "Creating new shard replicas" API right after bootstrapping a new cluster and loading data, but before clients start sending new requests?
+
+Yes, that's correct. Before clients start sending requests, you can use the "Creating new shard replicas" API to replicate all shards to the listener node. This ensures that all new data will be replicated to the data node, facilitating whole cluster backup or scaling out.
+
+### Does a listener node in a cluster receive data from all shards?
+
+Yes, a listener node can indeed receive data from all shards. However, it's important to note that a listener node doesn't participate in searches.
+
+###  When creating snapshots from the listener node, will they include data from all shards by default?
+
+No, snapshots from the listener node won't automatically include data from all shards. You'll need to manually replicate all shards to the listener node. You can use the `replicate_shard` API for this purpose.
+
+
+### If I want to use a listener node solely for data storage and scaling out my cluster, can I do so without affecting other nodes with query traffic?
+
+Absolutely. You can configure a listener node to act as a pure data node, thereby scaling out your cluster without impacting other nodes with query traffic.
```

Comment:
we are going to deprecate Listener node and it is definitely not a frequently asked question